OrbitClientModel/include/OrbitClientModel/CaptureSerializer.h

```diff
@@ -26,9 +26,18 @@ ErrorMessageOr<void> Save(const std::string& filename, const CaptureData& captur
 void WriteMessage(const google::protobuf::Message* message,
                   google::protobuf::io::CodedOutputStream* output);
 
+namespace file_management {
```

Comment:
I'd probably put the two functions below in the capture_serializer namespace. Unless I miss the reason for putting them in a nested namespace.

Reply:
I was not sure which of the two options was better so I decided to put it in there and see if there were any comments about it in the review :D I will move it to its own namespace, thanks!

Reply:
Ok, I re-read this and I am updating the comment. I wanted to make a distinction about being part of file_management because these methods do not really serialize the data and it would be confusing, but they are related to the data serialization. What it might make sense to do instead of a nested namespace is moving the namespace below so it is independent but also part of CaptureSerializer, would that make sense?

Reply:
I am thinking that it might also make sense to rename it to sth like _orbit_file_

Reply:
capture_serializer contains stuff that is related to serialization. The functions have proper names - so I don't think it is confusing at all. 
Having many nested namespaces unnecessarily complicates names (capture_serializer::file_management::IncludeOrbitExtensionInFile). As a general guidance: Often a namespace contains everything that is in a directory. Sometimes everything that is in a project. The reason why the stuff here is in a distinct namespace is that "Save" is a very generic name prone to collisions (and the fact that it used to be a class). It would have been completly fine to rename "Save" to "SaveCaptureToFile" and leave it in the top level namespace.
 
Another way of thinking of this: Namespaces are not so much used for naming things but to avoid collisions. (Ahh, that is not entirely true. But I think the direction is okay.)

Reply:
Ok, got it, thanks for the explanation :) Will update the code. Thanks!

Reply:
Done!